src/Microsoft.Identity.Client/AppConfig/AbstractApplicationBuilder.cs

```diff
@@ -105,6 +101,19 @@ internal T WithHttpManager(IHttpManager httpManager)
             return (T)this;
         }
 
+        /// <summary>
+        /// Use when the AAD admin has enabled conditional access. Acquiring the token normally will result in a
```

Comment:
"Use went the tenant admin has enabled".

"Acquiring a token (either in your app, or in a downstream API) could result in a"